core/src/repair/serve_repair.rs

```diff
@@ -229,7 +229,7 @@ type PingCache = ping_pong::PingCache<REPAIR_PING_TOKEN_SIZE>;
 #[cfg_attr(
     feature = "frozen-abi",
     derive(AbiEnumVisitor, AbiExample),
-    frozen_abi(digest = "9KN64WUT7XDYj9zZopS1hztGyAP9y4N4QznsyC4mqsGs")
+    frozen_abi(digest = "FGw38CCo7vg24qxe7TfGP11WdX2poe6T55BGN1r3XMFA")
```

Comment:
why does the hash change here? Was the hash algo changed somehow?

Reply:
wow I did not expect frozen_abi to care about filed names (as they are not part of the ABI). But ok I guess this is fine then=)